internal/cortex/chunk/cache/cache.go

```diff
@@ -5,10 +5,11 @@ package cache
 
 import (
 	"context"
-	"errors"
 	"flag"
 	"time"
 
+	"github.com/pkg/errors"
+
```

Comment:
Nit: this seems like an extra newline

Reply:
Actually not, `gofumptd` and `go fmt` adds this :smile: